appraisal/ruby-2.7.rb

```diff
@@ -193,6 +193,7 @@
   gem 'concurrent-ruby'
   gem 'dalli', '>= 3.0.0'
   gem 'grpc'
+  gem 'karafka'
```

Comment:
karafka no longer supports 2.7 based on: https://karafka.io/docs/Versions-Lifecycle-and-EOL/